script/check-collections.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const defaultCollections = require('../script/collections/default.json');
+const brandConsolidationCollections = require('../script/collections/brand-consolidation.json');
+
+const checkCollections = buildOptions => (files, metalsmith, done) => {
+  const collection = buildOptions['brand-consolidation-enabled']
+    ? Object.keys(brandConsolidationCollections)
+    : Object.keys(defaultCollections);
+
+  const brokenCollections = Object.keys(files).reduce((accum, key) => {
+    const file = files[key];
+
+    if (key.includes('.html')) {
+      if (file.collection.length > 0) {
+        file.collection.forEach(collectionName => {
+          if (!collection.includes(collectionName)) {
+            accum.push(
+              `Error: Your [ collection "${collectionName}" ] does not exist in the collection: ${key}`,
+            );
+          }
+        });
+      }
+
+      if (file.children && !collection.includes(file.children)) {
+        accum.push(
+          `Error: Your [ children: "${
+            file.children
+          }" ] does not exist in the collection: ${key}`,
+        );
+      }
+    }
+
+    return accum;
+  }, []);
+
+  if (brokenCollections.length > 0) {
+    console.warn(
```

Comment:
Not sure if this was the best way to log out the errors. If you guys know a better way definitely share.

Reply:
Seems reasonable to me